src/random_test.cpp

```diff
@@ -1638,11 +1696,35 @@ void Table::UpdateRandomROW(Thd1 *thd) {
       break;
     }
   }
+  if (prob == 0)
```

Comment:
use 1st suggestion for updates and selects.

Reply:
Ok